django_saml2_auth/views.py

```diff
@@ -183,6 +186,16 @@ def acs(r):
     else:
         return HttpResponseRedirect(get_reverse([denied, 'denied', 'django_saml2_auth:denied']))
 
+    if settings.SAML2_AUTH.get('USE_JWT') is True:
+        # We use JWT auth send token to frontend
+        jwt_token = jwt_encode(target_user)
+        query = '?uid={}&token={}'.format(target_user.id, jwt_token)
```

Comment:
1. Why do you pass the uid? It should be in the token and using it from elsewhere is not safe anyway.
2. If the frontend is a SPA, shouldn't you use a fragment identifier to pass the data rather than query parameters, i.e. `url_suffix = '#token={}'.format(jwt_token)`?